### DISCUSSION TITLE
Huge RMSE_E difference between train set to test & validation sets

### DISCUSSION THREAD

**Discussion by eren-ismail** [ACEsuit/mace#1025] [General]
Dear all,

I was started to use almost a year and half ago and I believe one of 2.X version. With the dataset I used, I get reasonable values for the error tables of the both stages. After I updated MACE to the latest version, error tables becomes like:

INFO: Error-table on TRAIN and VALID:
+---------------+---------------------+------------------+-------------------+
|  config_type  | RMSE E / meV / atom | RMSE F / meV / A | relative F RMSE % |
+---------------+---------------------+------------------+-------------------+
| train_Default |            9.9      |         50.3     |          5.57     |
| valid_Default |       796363.1      |         50.5     |          5.72     |
+---------------+---------------------+------------------+-------------------+
INFO: Error-table on TEST:
+-----------------+---------------------+------------------+-------------------+
|   config_type   | RMSE E / meV / atom | RMSE F / meV / A | relative F RMSE % |
+-----------------+---------------------+------------------+-------------------+
| Default_Default |       796420.6      |         54.1     |          6.05     |
+-----------------+---------------------+------------------+-------------------+

It might be just numerical issue because If I put the final on a molecular dynamic test, everything seems fine.

My question is that should I suspect anything? Is there any error related to my dataset?

Best,
Ismail

**Comment by ilyes319** [comment]
Hey @eren-ismail ,

Can you check the different keys in your xyz file you are using for your validation set and training set are the same?

**Reply by eren-ismail** [reply]
Hi Ilyes,

Thank you for your quick response. I checked all the keys in all of the xyz files and they are same.
To create my dataset, I combine all xyz files into one file, then randomly split %80 as train, %10 validation and %10 as test sets. I always use this procedure and test it with 6 different datasets. Resulting error-table of training shows similar pattern where RMSE_E for test and validation are higher, RMSE_E for train is small.

**Comment by eren-ismail** [comment]
I have solved the issue. It is related to the keys. if energy key is just 'energy' then it is ok for training but not for validation and test. So changing key to something else do the work.
Best
